coniferest/onnx/_parse.py

```diff
@@ -0,0 +1,50 @@
+from ._container import ConiferestModelContainer
+from onnxconverter_common.topology import Topology
+from onnxconverter_common.data_types import FloatTensorType
+
+from ..aadforest import AADForest
+
+
+def _get_coniferest_operator_name(model):
+    if isinstance(model, AADForest):
```

Comment:
What if we have a subclass, do we still want this if statement match?

Reply:
This is good question. ONNX-serialized model is just a forest. It doesn't know, and doesn't need to know how it was constructed. So, subclass might work if `score_samples` are not overridden.

Reply:
Than let's maybe restrict it to the specific class?